gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py

```diff
@@ -3995,3 +3995,15 @@ def impl(context):
     cmd =  "sudo mv -f /tmp/hosts_orig /etc/hosts; rm -f /tmp/clusterConfigFile-1; rm -f /tmp/hostfile--1"
     context.execute_steps(u'''Then the user runs command "%s"''' % cmd)
 
+@given('"LC_ALL" is different from English')
+def step_impl(context):
+	os.system('sudo locale-gen ru_RU.UTF-8 > /dev/null')
+	os.environ['LC_ALL'] = 'ru_RU.utf8'
```

Comment:
may be add before this row something like
```python
default_locale = os.environ.get('LC_ALL')
```
to ensure we save actual var instead one when file was imported?

Reply:
fixed